ibtk/src/utilities/HierarchyIntegrator.cpp

```diff
@@ -333,7 +333,7 @@ HierarchyIntegrator::getMinimumTimeStepSize()
     double dt = getMinimumTimeStepSizeSpecialized();
     for (const auto& child_integrator : d_child_integrators)
     {
-        dt = std::max(dt, child_integrator->getMinimumTimeStepSize());
+        dt = std::min(dt, child_integrator->getMinimumTimeStepSize());
```

Comment:
this should be `std::max()` not `std::min()`

Reply:
The minimum time step size will be the max over all of the minimums.

The maximum time step size will be the min over all of the maximums.

Reply:
Yes, I see. This isn't choosing smallest stable time steps, it's choosing minimum timesteps.

Reply:
I'm pretty sure this is the second time I've gotten confused on this :laughing: